### PR TITLE
chg: [i18n] Added a lot of __('s for our i18n effort

### PR DESCRIPTION
**Pull request by SteveClement**
new: [i18n] Added initial Czech translation
chg: [i18n] Updated various translation to their latest version
chg: Wrapped 'command_line_functions' and ServerSettings into: public function __construct($id = false, $table = null, $ds = null) {
chg: [doc] Added Kali Linux install script. Working on latest. (W25)

### CODE REVIEW

File: app/Controller/ShadowAttributesController.php [hunk 1]
```diff
@@ -697,7 +697,7 @@ public function edit($id = null) {
 					foreach ($this->ShadowAttribute->validationErrors as $k => $v) {
 						$message .= '[' . $k . ']: ' . $v[0] . PHP_EOL;
 					}
-					throw new NotFoundException('Could not save the proposal. Errors: ' . $message);
+					throw new NotFoundException(__('Could not save the proposal. Errors: ', $message));
```
Comment (iglocska) [inline app/Controller/ShadowAttributesController.php]: This is incorrect, missing the %s after "Errors: "

File: app/View/Elements/healthElements/tabs.ctp [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="tabMenuFixedContainer">
 	<span class="tabMenuFixed tabMenuFixedLeft tabMenuSides">
-	<a href="<?php echo $baseurl;?>/servers/serverSettings/" id="create-button" title="<?php echo __('Modify filters');?>" class="discrete">Overview</a>
+  <a href="<?php echo $baseurl;?>/servers/serverSettings/" id="create-button" title="<?php echo __('Modify filters');?>" class="discrete"><?php echo __('Overview'); ?></a>
```
Comment (iglocska) [inline app/View/Elements/healthElements/tabs.ctp]: Indentation?

File: INSTALL/INSTALL.debian9.txt [hunk 3]
```diff
@@ -606,6 +606,7 @@ cd misp-dashboard
 sudo /var/www/misp-dashboard/install_dependencies.sh
 sudo sed -i "s/^host\ =\ localhost/host\ =\ 0.0.0.0/g" /var/www/misp-dashboard/config/config.cfg
 sudo sed -i -e '$i \sudo -u www-data bash /var/www/misp-dashboard/start_all.sh\n' /etc/rc.local
+sudo sed -i '/Listen 80/a Listen 0.0.0.0:8001' /etc/apache2/ports.conf
```
Comment (iglocska) [inline INSTALL/INSTALL.debian9.txt:609]: Should be part of a separate PR.

**Review by iglocska — COMMENTED** [review]
The install directory changes are unrelated to the internationalisation changes. Some minor other comments.

File: app/Controller/ShadowAttributesController.php (hunk 1 above)
Comment (SteveClement) [reply app/Controller/ShadowAttributesController.php]: f'in Typo ;)
Thanks.

File: INSTALL/INSTALL.debian9.txt (hunk 3 above)
Comment (SteveClement) [reply INSTALL/INSTALL.debian9.txt:609]: #cherry pick…

**Comment by iglocska** [comment]
Thanks a lot!
